lib/game.rb

```diff
@@ -28,6 +28,22 @@ def self.lowest_total_score
     all_scores.min
   end
 
+  def self.percentage_home_wins
+    home_wins = @@all.find_all { |game| game.home_goals > game.away_goals}.count
+    ( home_wins.to_f / @@all.length.to_f ).round(1) * 100
+  end
+
+  def self.percentage_visitor_wins
+    visitor_wins = @@all.find_all { |game| game.home_goals < game.away_goals}.count
+    ( visitor_wins.to_f / @@all.length.to_f ).round(1) * 100
+  end
+
+  def self.percentage_ties
+    ties = @@all.find_all { |game| game.home_goals == game.away_goals}.count
+    ( ties.to_f / @@all.length.to_f ).round(1) * 100
```

Comment:
lots of code shared between the home_wins, visitor_wins, and ties. Seems like a good opportunity for refactor once we get everything functional.